Source/Widgets/Viewer/Viewer.js

```diff
@@ -608,6 +614,20 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
 
         cesiumWidget.screenSpaceEventHandler.setInputAction(pickAndSelectObject, ScreenSpaceEventType.LEFT_CLICK);
         cesiumWidget.screenSpaceEventHandler.setInputAction(pickAndTrackObject, ScreenSpaceEventType.LEFT_DOUBLE_CLICK);
+
+        var queryOptions = queryToObject(window.location.search.substring(1));
```

Comment:
Core Cesium should never look at the app-specific window.location, or react to values there, because we won't know what any given app might be doing with its URL routing.  Instead, use [Apps/CesiumViewer](https://github.com/AnalyticalGraphicsInc/cesium/blob/1.9/Apps/CesiumViewer/CesiumViewer.js#L38) to get this value and pass it into the geocoder from there.